action.yaml

```diff
@@ -19,5 +19,5 @@ runs:
     - id: read-yaml
       run: |
         VERSION=`cat ${{inputs.file-path}} | grep -o 'version:[^:]*' | cut -f2 -d":" | xargs`
-        echo "::set-output name=version::$(echo $VERSION)"
+        echo "version=$VERSION" >> "$GITHUB_OUTPUT"
```

Comment:
I didn't try it, Bit don't you have to Define the Names of the output in the configuration of the Job, as described here: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs ?

Reply:
Hm no i didn't have to do it, it worked well this way, but i didn't try it recently either, maybe there was an upgrade, i'll look into it, thanks